Logs.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <div class="container">
+        <div class="logs-container">
+            <h2>Monday-Upper</h2>
+            <div class="logs-list" id='upperData'>
+                <ul>
+                    <li v-for="session in trainingSessions" key:"session.id"> monday:{{session.mondayData}} </li>
+                </ul>
+            </div>
+        </div>
+        
+        <div class="logs-container">
+            <h2>Tuesday-Lower</h2>
+            <div class="logs-list">
+                <ul>
+                    <li>Push press</li>
+                    <li>Incline Bench</li>
+                    <li>Lat pull downs</li>
+                    <li>Incline rows</li>
+                    <li>Face pulls</li>
+                    <li>Banded push downs</li>
+                </ul>
+            </div>
+        </div>
+        <div class="logs-container">
+            <h2>Thursday-Fluff</h2>
+            <div class="logs-list">
+                <ul>
+                    <li>Push press</li>
+                    <li>Incline Bench</li>
+                    <li>Lat pull downs</li>
+                    <li>Incline rows</li>
+                    <li>Face pulls</li>
+                    <li>Banded push downs</li>
+                </ul>
+            </div>
+        </div>
+        <div class="logs-container">
+            <h2>Saturday</h2>
+            <div class="logs-list">
+                <ul>
+                    <li>Push press</li>
+                    <li>Incline Bench</li>
+                    <li>Lat pull downs</li>
+                    <li>Incline rows</li>
+                    <li>Face pulls</li>
+                    <li>Banded push downs</li>
+                </ul>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+       let itemDataUpper = new Vue({
+        el: '#upperData',
+        mondayData() {
+            return {
+            trainingSessions: [
+                     {monday: 'Strict Press'},
+                     {monday: 'Incline Bench Press'},
+                     {monday: 'Lat Pull Downs'},
+                     {monday: 'Banded Face Pulls'},
+                     {monday: 'Banded Push Downs'},
+                     {monday: 'One Arm Row'},
+                     {monday: '3-way Raise'},
+                         ] 
+            }
+         }
+    });
+    </script>

```

Comment:
This is ok for learning purposes but get in the habit of using Single File Components. So you'll have a root vue entry for your app, where you'll attach Vue. For regular components you can use [ SFC ](https://vuejs.org/v2/guide/single-file-components.html) syntax